### PR TITLE
add docker compose file

### PR DESCRIPTION
**Pull request by pfefferle**
to simplify local testing

> To test the plugin on your local machine, simply run `docker-compose up` in the root folder and open `http://localhost:8081/` in your browser.

### CODE REVIEW

**Comment by pfefferle** [comment]
@2ndkauboy friendly ping ;)

**Comment by 2ndkauboy** [comment]
Works perfectly even with WSL :)

**Review by 2ndkauboy — APPROVED** [review]

**Comment by pfefferle** [comment]
Thanks @2ndkauboy

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Say Goodbye to comment spam on your WordPress blog or website. *Antispam Bee* bl
 * Pull requests for documented bugs are highly appreciated.
 * If you think you’ve found a bug (e.g. you’re experiencing unexpected behavior), please post at the [support forums](https://wordpress.org/support/plugin/antispam-bee) first.
 * If you want to help us translate this plugin you can do so [on WordPress Translate](https://translate.wordpress.org/projects/wp-plugins/antispam-bee).
+* To test the plugin on your local machine, simply run `docker-compose up` in the root folder and open `http://localhost:8081/` in your browser.
```
Comment (2ndkauboy) [inline README.md:35]: @pfefferle Maybe write that they should use `docker-compose up -d` to run it in "detached mode"?

**Review by 2ndkauboy — CHANGES_REQUESTED** [review]

**Pull request merged**